contrib/checkbox-ce-oem/checkbox-provider-ce-oem/bin/cold_reboot_by_pdu.sh

```diff
@@ -5,6 +5,58 @@ outlet_port=""
 pdu_type=""
 reboot_duration=""
 
+cold_reboot_raritan() {
+    echo "Turn off the power of DUT by Raritan network PDU.."
+    echo "System should be back after around $reboot_duration seconds.."
+    echo ""
+    # change duration of power reboot
+    oid=".1.3.6.1.4.1.13742.6.3.5.3.1.12.1.$outlet_port"
```

Comment:
May I know where does these numbers come from?

Reply:
the SNMP OID comes from Raritan website
https://cdn1.raritan.com/download/pdu-g2/4.2.0/PDU2_MIB_4.2.0_50274.txt